doc/developers-guide/domain_management.md

```diff
@@ -118,10 +118,120 @@ The number of seconds after an event must be deleted from the `domain_events` ta
 
 # REST API
 
-We must provide REST API for Add/Remove and Enable/Disable interfaces of the
-MongooseIM service described in the section above.
+Provides API for adding/removing and enabling/disabling domains over HTTP.
 
-# Command Line Interfaces
+Implemented by `mongoose_domain_handler` module.
+
+Configuration example:
+
+```toml
+[[listen.http]]
+  ip_address = "127.0.0.1"
+  port = 8088
+  transport.num_acceptors = 10
+  transport.max_connections = 1024
+
+  [[listen.http.handlers.mongoose_domain_handler]]
+    host = "localhost"
+    path = "/api"
+```
+
+## Add domain
+
+```bash
+curl -v -X PUT "http://localhost:8088/api/domains/example.db" \
+    -H 'content-type: application/json' \
+    -d '{"host_type": "type1"}'
+```
+
+Result codes:
+
+* 204 - inserted.
+* 403 - domain already exists with a different host type.
```

Comment:
I think `rest_cannot_insert_domain_twice_with_another_host_type` expects code 409 in this case.